README.md

```diff
@@ -371,6 +371,20 @@ if let foo = foo {
 }
 ```
 
+Or when unwrapping multiple optionals, prefer this:
+
+```swift
+if 
+    let foo = foo,
```

Comment:
Actually with such short if lets it could all be on one line, IMO. But for longer ones it should be split up.
```swift
if let foo = foo, let bar = bar {
    foo.go()
}
// vs.
if 
    let fooProp = foo.optionalPropertyOnFoo as? SomeType,
    let bar = bar as? OtherType {
        foo.go()
        bar.stop()
}
```